.github/workflows/ci.yml

```diff
@@ -444,25 +452,25 @@ jobs:
           # Setup Arch and Target
           $target = $apiLevel -ge 30 ? 'google_apis' : 'default'
           Write-Output "Current Target is $target"
+          "target=$target" >> $env:GITHUB_OUTPUT
           "api-level=$apiLevel" >> $env:GITHUB_OUTPUT
           "label=$($label ?? $apiLevel)" >> $env:GITHUB_OUTPUT
-          "target=$target" >> $env:GITHUB_OUTPUT
 
-      - name: Android API ${{ steps.droid-settings.outputs.label }} emulator setup + Smoke test
-        id: smoke-test
-        timeout-minutes: 15
+      - name: Android API ${{ steps.config.outputs.label }} emulator setup + Smoke test
         uses: reactivecircus/android-emulator-runner@76c2bf6f95ed6458fd659a1bdb680a0f8df232dc
```

Comment:
You may want to bump the emulator action to the current version `50986b1464923454c95e261820bc626f38490ec0`

Reply:
```suggestion
        uses: reactivecircus/android-emulator-runner@50986b1464923454c95e261820bc626f38490ec0
```

Reply:
Thanks, it wasn't out yet when I last updated this PR